MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+> maintainers.
+>
+> Maintainers will take into consideration if a patch is in line with the general
+> principles of the project; meets the minimum standards for inclusion; and will
+> judge the general consensus of contributors.
+
+Thus maintainers role is not directly elevated above that of regular
+contributors, as maintainers serve mostly as arbiters of if changes are
+acceptable to the contributors to the project in aggregate.
+
+The role of the Maintainer is currently evolving with respect to "Scoped
+Maintainers". The concept of a scoped maintainer is that it is helpful to add
+maintainers who are subject matter experts in a particular area of the code,
+but perhaps not in other areas. As such, maintainers with an explicit scope
+typically will avoid merging PRs in areas outside their scope, but may do so on
+occasion. As an individual maintainer's experience levels in the project
+increase, they may change their focus from scoped to general, but this is not
+currently an explicit designation (all maintainers should be considered to be
+general at this time). As a rough measure, scoped maintainers should avoid
+"stepping on the toes" of work that is being guided by another maintainer or
+contributor.
+
+Historically, Bitcoin has also had a "Lead Maintainer", which was most recently
+@laanwj. However, [in a blog
+post](https://laanwj.github.io/2021/01/21/decentralize.html), @laanwj stated
+that they wanted to take more of a background role as a maintainer and have not
+"passed the baton" to a successor as a Lead Maintainer, and instead focused on
+decentralizing the maintainership ecosystem. As such, while the title of Lead
+Maintainer still is held by @laanwj, it should be considered a bug-not-feature
+if it confers any additional responsibility or privilege over other General
+Maintainers, and may be explicitly deprecated in the future.
+
+## Self Merges
+
+It is discouraged for maintainers to merge their own work. However, given that
+there may not be another maintainer who regularly works on the section of code
+or is familiar, self-merge is not forbidden. However, as a guard, self merges
+should usually only happen if the following criteria has been met:
+
+1. The PR has been reviewed and ACK'd by other regular contributors familiar
+   with that section of the code.
+1. There are no NACKs being overruled on that PR.
+1. The PR is at least 1 week old, to permit time for review.
+1. The maintainer should communicate intent to self-merge 48 hours before doing so
+
+These rules aren't hard and fast, but are considered a best practice to avoid
+self-merge as a convenience.
+
+
+
+## Nomination and Selection of Maintainers
+
+Being a Maintainer is not an honor (but it may be honorable to serve as one).
+Many regular contributors who are capable or experienced enough choose not to
+be a Maintainer. Similarly, just because a regular contributor has reached a
+certain level of experience does not mean that they "deserve" to be a maintainer.
```

Comment:
Can this whole paragraph be removed? It reads more like an editorial, and all the actual info is in the following paragraphs anyway (also, what's all that stuff about honor? Did a Klingon write this?).

Reply:
I mean strictly speaking this paragraph doesn't explain what a maintainer is, just what a maintainer is _not_ (and who did _not_ become a maintainer). I guess what the paragraph is trying to convey is that it isn't some career goal that you obtain when you reach Bitcoin programmer level 35. Which is fine to mention, but at least I wouldn't open the section with that. And if at all possible I would avoid words like "honour" in a technical role description (maybe replace with "achievement award" or similar?), but maybe that's just me.

Reply:
it took me a while to find this (I blame british/american spelling differences). This is the reason why I included the bit about honor/honour in particular.

https://www.coindesk.com/markets/2014/04/08/gavin-andresen-steps-down-as-bitcoins-lead-developer/

> "I had noticed that Gavin had been much less active in the Github project lately, but I did not expect him to step down as lead developer. But it makes sense, there is a lot more theoretical work with regard to Bitcoin and cryptocurrency in general making it a full-time job just to keep up with that (and giving talks, travelling, and such). On the other hand, I have been effectively the maintainer of the code for quite a while. In practice nothing changes."
He added that it was "an honour to be entrusted with the role" and said the positive responses he had received were a reminder that his work was appreciated in the community: "it's easy to forget that sometimes with all the trolling and meanness going on around Bitcoin!"


I felt it is important to clarify that it's not the type of honor that is like an award, even though there is honor in it.

Reply:
To me, "there is honor in it" sounds like something a politician would say to get young people to do something stupid (like invade some foreign country), but I guess we were just socialised differently. But before I stop beating that dead horse, let me point out that the quote you posted directly contradicts your proposed text:

> He added that it was "an honour to be entrusted with the role" 

vs.

> Being a Maintainer is not an honor

(and doing something honorably is not the same thing as feeling honored to be entrusted with something. But again, I'll stop now, our usage of/associations with that word are just different I guess)

Reply:
it's fine to discuss and i'm interested to meaningfully incorporate your feedback.

perhaps one of:

> Being a Maintainer is not a bestowed honor (but it may be honorable to serve as one).

> Being a Maintainer is not an honorific title (but it may be honorable to serve as one).

> Being a Maintainer is not an "honorable status" (but it may be honorable to serve as one).

> Being a Maintainer is not an "honorable appointment" (but it may be honorable to serve as one).

Paired with one of:

> (but it may be honorable to serve as one).

> .

> , not to be confused with the honorability of serving in this role which is beyond the scope of this document.